public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx

```diff
@@ -80,13 +79,6 @@ export const SingleTopBar = memo(function SingleTopBar({
             <ToolbarButton iconOnly icon="question-circle" aria-label="Help" />
           </Dropdown>
         )}
-        {config.newsFeedEnabled && <NewsContainer />}
-        <ToolbarButton
-          icon="monitor"
-          className={styles.kioskToggle}
-          onClick={onToggleKioskMode}
-          tooltip="Enable kiosk mode"
-        />
```

Comment:
did you mean to also get rid of the Kiosk mode button?

Reply:
no that was not my intention! thanks for saving me from that embarrassment. fixed!